Source/WebCore/style/StyleResolver.cpp

```diff
@@ -340,9 +340,6 @@ ResolvedStyle Resolver::styleForElement(Element& element, const ResolutionContex
     Adjuster adjuster(document(), *state.parentStyle(), context.parentBoxStyle, &element);
     adjuster.adjust(style, state.userAgentAppearanceStyle());
 
-    if (style.usesViewportUnits())
-        document().setHasStyleWithViewportUnits();
-
```

Comment:
Do we have cases where StyleAdjuster would use viewport units that `applyMatchedProperties` would not cover?

Reply:
I don't think so.